### PR TITLE
Duplicate Jenkinsfiles for Uyuni until we can use parameters

### PR DESCRIPTION
**Pull request by juliogonzalez**

### CODE REVIEW

File: jenkins_pipelines/uyuni_prs/tests_files/rubocop.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#! /bin/bash
```
Comment (MalloZup) [inline jenkins_pipelines/uyuni_prs/tests_files/rubocop.sh]: @juliogonzalez  if we go opensource and the repo is public, we might run this via travis, i think we could consider it.
in

Comment (juliogonzalez) [reply jenkins_pipelines/uyuni_prs/tests_files/rubocop.sh]: IMHO we should use the same stuff for both Uyuni and SUSE manager, to avoid confusing people.

Comment (MalloZup) [reply jenkins_pipelines/uyuni_prs/tests_files/rubocop.sh]: ok

**Comment by juliogonzalez** [comment]
Thanks! Merging now.

**Pull request merged**